python/sparkdl/utils/jvmapi.py

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
The `pyutils` is the one from spark's [PythonUtils](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/api/python/PythonUtils.scala).